pkg/storage/db.go

```diff
@@ -1003,3 +1024,17 @@ order by ae.ts asc
 	}
 	return nil
 }
+
+func isPublicIP(ip net.IP) bool {
+	privateIPBlocks := make([]*net.IPNet, 0, len(privateIPs))
+	for _, privateIP := range privateIPs {
+		_, block, _ := net.ParseCIDR(privateIP)
+		privateIPBlocks = append(privateIPBlocks, block)
+	}
+	for _, block := range privateIPBlocks {
+		if block.Contains(ip) {
+			return false
+		}
+	}
+	return true
+}
```

Comment:
not crazy about this function having to parse the same array of private IP addresses every single time a read comes in for the new schma but wasn't sure how else to do this without some sort of init() function. Open to suggestions :)